walk-in-interview/src/main/java/com/google/job/data/Requirement.java

```diff
@@ -56,4 +73,13 @@ public String getLocalizedName(String language) throws IllegalArgumentException
 
         return localizedNames.build();
     }
+
+    /**
+     * Returns the localized names of the all requirements.
+     *
+     * @throw IllegalArgumentException If the language is not supported.
+     */
+    public static List<String> getAllLocalizedNames(String language) throws IllegalArgumentException {
```

Comment:
again, isn't getLocalizedName() enough?

Reply:
in fact, i am considering if we need to store localized name in database... if we don't need it at all, then there is no need to localized name field for requirement; but i will leave it there for now